src/app/features/cases/components/Workflow/WorkflowForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+import { ScaffoldForm } from "@amsterdam/amsterdam-react-final-form"
+
+import ScaffoldFields from "app/features/shared/components/molecules/Form/ScaffoldFields"
+import { Spinner } from "@amsterdam/asc-ui"
+import ConfirmScaffoldFields from "app/features/shared/components/molecules/ConfirmScaffoldFields/ConfirmScaffoldFields"
+import useSubmitConfirmation from "app/features/shared/components/molecules/ConfirmScaffoldFields/hooks/useSubmitConfirmation"
+
+type Props = {
+  caseId: Components.Schemas.Case["id"]
+  endpoint: any
+  scaffold: any
+  extraLabel?: string
+}
+
+const WorkflowForm: React.FC<Props> = ({ caseId, scaffold, endpoint, extraLabel }) => {
+
+  const result = endpoint()
+
+  //TODO allow for PUT
+  const { execPost } = endpoint()
```

Comment:
I would inject a `postMethod` parameter, instead of `endpoint`. Then you can also inject either post / put / delete.

Reply:
I think I need both then, see my last commit. Or can I combine endpoint and postMethod?

Reply:
Refactored it